test.js

```diff
@@ -43,23 +44,18 @@ function test (label, options, fn) {
 
     var sourceData = ctx.sourceData = []
     for (var i = 0; i < 10; i++) {
-      ctx.sourceData.push({ type: 'put', key: i, value: Math.random() })
+      ctx.sourceData.push({ type: 'put', key: String(i), value: 'value' })
     }
 
     ctx.verify = function (ws, done, data) {
-      // can pass alternative data array for verification
-      data = data || sourceData
-      var _done = after(data.length, done)
-      data.forEach(function (data) {
```

Comment:
This is a first step in changing data to be more than 10 elements. Updating to e.g. 1000 elements might help out with reproducing the race conditions and we can't have 1000 gets and 1000 comparisons.

Reply:
Investigating this further today.

Reply:
Hmm interesting. It seems that if I _reduce_ the data it increases the chance of race condition, i.e. if the batch size is really small, like a single elements to a couple of elements. If I make sure `ctx.sourceData` is a magnitude larger (100 elements)